src/regression_tracker.py

```diff
@@ -31,6 +31,17 @@ def _stop(self, sub_id):
         if sub_id:
             self._api_helper.unsubscribe_filters(sub_id)
 
+    def _collect_logs(self, node):
+        """Returns a dict containing the log artifacts of <node>. Log
+        artifacts are those named 'log' or whose name contains the
+        '_log' suffix.
+        """
+        logs = {}
+        for artifact, value in node['artifacts'].items():
+            if artifact == 'log' or '_log' in artifact:
```

Comment:
```suggestion
            if artifact == 'log' or artifact.endswith('_log'):
```

This is a bit stricter and more in line with the function docstring.